core/forwarder.py

```
@@ -224,6 +237,102 @@ def _is_spoiler_message(msg: Message) -> bool:
+
+        if self._message_age_seconds(messages[-1]) < settle_seconds:
+            tail_start_index = len(messages) - 1
+            while (
+                tail_start_index > 0
+                and getattr(messages[tail_start_index - 1], "grouped_id", None)
+                == grouped_tail
+            ):
+                tail_start_index -= 1
+            stable_messages = messages[:tail_start_index]
+            logger.info(
+                f"[Merge] 频道 {channel_name} 最新相册 {grouped_tail} 仍在到达窗口内，暂缓 {len(messages) - len(stable_messages)} 条。"
```

Ruff (0.15.20)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)